README.md

```diff
@@ -37,3 +37,60 @@ SpiNNaker.  Tools exist for:
  - Clock discipline
  - Routing table minimisation
  - Optimal routing key allocation
+
+## Developing Rig
+
+### Installing in a `virtualenv`
+
+Create a new [virtualenv](https://pypi.python.org/pypi/virtualenv) by running:
+
+    virtualenv new_directory_name
```

Comment:
Could you add the arguments that keep your local numpy etc? (/me cannot remember this...)